addons/point_of_sale/models/pos_config.py

```diff
@@ -580,7 +580,13 @@ def post_install_pos_localisation(self, companies=False):
         if not companies:
             companies = self.env['res.company'].search([])
         for company in companies.filtered('chart_template'):
-            pos_configs = self.search([('company_id', '=', company.id)])
+            pos_configs = self.search([('company_id', '=', company.id), ('module_pos_restaurant', '=', False)])
+            if not pos_configs:
+                self = self.with_company(company)
+                pos_configs = self.env['pos.config'].create({
+                'name': 'Shop',
+                'company_id': company.id,
```

Comment:
You maybe should look if module_pos_restaurant is not True by default (I'm not so sure but if it is the case, you need to set it to False for this pos)